viewer/src/views/threejs/cad/CadNode.ts

```diff
@@ -177,7 +176,7 @@ export class CadNode extends THREE.Object3D {
         fromThreeCameraConfig(),
 
         // Determine all wanted sectors
-        this._sectorCuller.determineSectors(),
+        map(input => this._sectorCuller.determineSectors(input)),
```

Comment:
This is according to what we agreed, which was to use non-RxJS implementations as far as possible. In other words, this change should be kept even if the others are rejected.